src/compiler/scanner.ts

```diff
@@ -429,6 +429,7 @@ namespace ts {
               case CharacterCodes.slash:
                   // starts of normal trivia
               case CharacterCodes.lessThan:
+              case CharacterCodes.bar:

```

Comment:
@alfaproject can you open a separate issue or PR for this? This PR doesn't change the `CharacterCodes` enum.